app/src/main/java/org/coepi/android/MainActivity.kt

```diff
@@ -34,11 +37,15 @@ class MainActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         setContentView(activity_main)
         observeRootNavigation()
+        Log.i(TAG_BLE_LOG, "Staring BLE");
         blePreconditions = BlePreconditions(this) {
             bleDiscovery.discover()
-            log.i("BlePreconditions met - BLE discover process started")
+            val ok= "BlePreconditions met - BLE discover process started";
+            log.i(ok)
+            Log.i(TAG_BLE_LOG, ok);//DONE: BARTPROBLEM on my phone (sdk=21) I don see ok! Now I see
```

Comment:
Cleanup before opening PR.

Reply:
no, there's a problem  i never see executing this
Log.i(TAG_BLE_LOG, ok);